README.md

```diff
@@ -95,6 +95,28 @@ We'll insert the following middlewares into the rails stack:
 2. `Rack::SslEnforcer`: enforces HTTPS.
 3. `Rack::Deflater`: compresses responses from the application, can be disabled with `ROO_ON_RAILS_RACK_DEFLATE` (default: 'YES').
 
+### Sidekiq
+
+Unless `SIDEKIQ_ENABLED` is set to false we'll check for the existance of sidekiq gems. Please ensure the following lines are included in your project Gemfile.
+
+```
+source 'https://rubygems.org' do
+  gem 'sidekiq-scheduler'
+  gem 'sidekiq-unique-jobs'
+end
+source 'https://gems.contribsys.com/' do
+  gem 'sidekiq-pro'
+end
+```
+
+Sidekiq pro requires some authentication to be installed. Please get the key from your Tech Lead (or password manager of choice).
```

Comment:
Did we decide that? Any links to the guidelines that we wrote on deliveroo.engineering 😉 ?